### ISSUE TITLE
Unable to delete server on tvOS

### ISSUE THREAD

**Issue by mreid-tt** [jellyfin/Swiftfin#1221]
### Describe the bug

Unlike in the iOS version when I long-press a server in the server list in tvOS, I don't seem to be presented with an option to remove the server from the list.

The steps I tried were as follows:
1. Click the settings (gear icon)
2. Click on switch user
3. Click the server
4. Long-press to remove the user
5. Go back to server list
6. Long-press to remove the server

### Application version

1.0.1 (70)

### Where did you install the app from?

App Store

### Device information

Apple TV 4K 64GB (3rd generation)

### OS version

tvOS 17.6.1 (21M80)

### Jellyfin server version

10.9.7

**Comment by S-en-B** [comment]
Have this issue as well. Same conditions, same steps. Any solution/resolution would be awesome

**Comment by arvet333** [comment]
You need to reinstall app to delete server

**Comment by mreid-tt** [comment]
> You need to reinstall app to delete server

While the workaround is helpful, the feature still isn't functioning as expected. I noticed that the iOS version is 1.1, whereas the tvOS version is still 1.0.1. It’s possible this issue has already been fixed in iOS, but the updated tvOS version may not have been released yet.

**Comment by whiterock** [comment]
Same here. I keep reinstalling the app.

**Comment by mreid-tt** [comment]
I noticed that the app has been updated to v1.2 in the main repository, but the tvOS version hasn't been updated. While I see the new version deployed to iOS and iPadOS, tvOS remains on the old version. Could you provide an update on when all versions will be aligned?

**Comment by wigsgiw** [comment]
> I noticed that the app has been updated to v1.2 in the main repository, but the tvOS version hasn't been updated. While I see the new version deployed to iOS and iPadOS, tvOS remains on the old version. Could you provide an update on when all versions will be aligned?

https://github.com/jellyfin/Swiftfin/discussions/1294

**Comment by JPKribs** [comment]
As a note, this is possible on the current Main branch of Swiftfin. Additionally, individual users can be deleted via https://github.com/jellyfin/Swiftfin/pull/1359

This Issue will remain open until this functionality is available from the App Store. Our tvOS version still needs some work before it is in a shippable state which can be tracked here: https://github.com/jellyfin/Swiftfin/discussions/1294

**Comment by samglt** [comment]
Is it still possible to delete servers on the current git version in either iOS or tvOS? I see no use of the `editServerFromSelectUser` route anywhere, which I assume is the only way to get to `EditServerView` with the `isEditing` environment variable set. It seems like [this route in `ServerSelectionMenu`](https://github.com/jellyfin/Swiftfin/blob/1b07876db47e013e2257bb51377510b19f4a983f/Swiftfin/Views/SelectUserView/Components/ServerSelectionMenu.swift#L47) should be `editServerFromSelectUser` instead of `editServer`

**Comment by JPKribs** [comment]
> Is it still possible to delete servers on the current git version in either iOS or tvOS? I see no use of the `editServerFromSelectUser` route anywhere, which I assume is the only way to get to `EditServerView` with the `isEditing` environment variable set. It seems like [this route in `ServerSelectionMenu`](https://github.com/jellyfin/Swiftfin/blob/1b07876db47e013e2257bb51377510b19f4a983f/Swiftfin/Views/SelectUserView/Components/ServerSelectionMenu.swift#L47) should be `editServerFromSelectUser` instead of `editServer`

Yes. That appears to have been broken during the large migration from `NavigationView` to `NavigationStack`. That will need to be resolved prior to our final 1.4 release.

**Comment by JPKribs** [comment]
> That will need to be resolved prior to our final 1.4 release.

Additionally, iOS Sign Out takes you back to the User Select while tvOS takes you to Server Select. That will need to be updates as well so they both go to the same User Select as with pre-`NavigationStack` routing

I'm sure we'll find more of these now we have our TestFlight out in the wild. The navigation changes were long-needed but it was a very very large PR so I suspect I missed some routing issues when I was helping test. We should be able to resolve those now 1.4 is out in the wild. These are the only 2 I am aware of.
